sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -1222,18 +1220,15 @@ private void notifyChannelConnect(final ChannelFuture future, final Promise<Chan
                         return channel;
                     });
                 } else {
-                    if (logger.isDebugEnabled()) {
-                        logger.debug("notifyChannelConnect promise.trySuccess(channel)=false");
-                    }
+                    logger.info("notifyChannelConnect local {}, remote {} promise.trySuccess(channel)=false", channel.localAddress(), channel.remoteAddress());
 
                     // Promise was completed in the meantime (like cancelled), just close the channel
                     this.closeChannel(channel);
                 }
 
             } else {
-                if (logger.isDebugEnabled()) {
-                    logger.debug("notifyChannelConnect future was not successful");
-                }
+                logger.error("notifyChannelConnect future was not successful {}", future.cause());
```

Comment:
do we need to log this as error if it was not logged before as such? how often this happens and is it logged in some other way?